test/integration/visitor_cannot_view_orders_and_admin_views_test.rb

```diff
@@ -72,13 +72,13 @@ class VisitorCannotViewOrdersAndAdminViewsTest < ActionDispatch::IntegrationTest
     fill_in 'Password', with: 'admin_password'
     click_button 'Login'
 
-    visit "/admin/dashboard/#{admin.id}"
+    visit '/admin/dashboard'
 
     assert page.has_content?('Welcome, Admin!')
 
     click_link('Logout')
 
-    visit "/admin/dashboard/#{admin.id}"
+    visit '/admin/dashboard'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.